spec/unit/query_components/highlight_spec.rb

```diff
@@ -2,36 +2,36 @@
 
 RSpec.describe QueryComponents::Highlight do
   describe "#payload" do
-    it "enables highlighting on title" do
+    it "enables highlighting on title with synonyms" do
       parameters = Search::QueryParameters.new(parsed_query: {}, return_fields: %w[title_with_highlighting])
 
       payload = described_class.new(parameters).payload
 
-      expect(payload[:fields].keys).to include(:title)
+      expect(payload[:fields].keys).to include("title.synonym")
```

Comment:
It's slightly weird that we sometimes use `"title.synonym".to_sym` (as in the previous commit)

Reply:
Ah, and I see that those bits have been removed in this commit